src/taxonomy/factory.ts

```diff
@@ -0,0 +1,32 @@
+import { v4 as uuidv4 } from 'uuid'
+
+import { Factory } from 'src/common'
+import { Taxonomy, TaxonomyProps } from './taxonomy'
+
+export type TaxonomyFactoryParams = {
+  props?: TaxonomyProps
+  draft?: boolean
+  published?: boolean
+}
+
+export const TaxonomyFactory: Factory<Taxonomy> = {
+  createInstance: (params: TaxonomyFactoryParams): Taxonomy => {
+    const defaultProps = {
+      props: {},
+      published: undefined,
+      draft: undefined,
```

Comment:
let's not leave undefined here.
when creating objects, they are draft and non published by default